text/src/autogluon/text/automm/data/randaug.py

```diff
@@ -0,0 +1,269 @@
+# code in this file is adpated from rpmcruz/autoaugment
```

Comment:
Is it possible to remove this file and reuse https://pytorch.org/vision/main/generated/torchvision.transforms.RandAugment.html?

Reply:
This file is copied from ViLT's implementation, which has removed two operations: color inversion and cutout. They give the reason of doing so in their paper:

"We use all the original policies except two: color inversion, because texts often contain color information as well, and cutout, as it may clear out small but important objects dispersed throughout the whole image."